flink-contrib/flink-storm-compatibility/flink-storm-compatibility-examples/src/main/java/org/apache/flink/stormcompatibility/wordcount/StormWordCountLocal.java

```diff
@@ -67,8 +67,10 @@ public static void main(final String[] args) throws Exception {
 
 		Utils.sleep(10 * 1000);
 
-		// TODO kill does no do anything so far
 		cluster.killTopology(topologyId);
+		// killing sends STOP signal, takes some time to clean up
+		Utils.sleep(1000);
```

Comment:
Unfortunately, we need to delay a little bit... I am not a fan of it either. However, I rebase this PR multiple times and it was never an issue on Travis (according test stability). If you don't have strong objection against it, I would leave it. (The main cause of the problem comes from Storm behavior we mimic here. `killTopology(...)` does not block in Storm, so we do not block here either.)